docs/how-to/use_guardrails.md

```diff
@@ -0,0 +1,51 @@
+# How-To: Use Guardrails
+
+ragbits offers an expandable guardrails system. You can use one of the available guardrails or create your own to prevent toxic language, PII leaks etc.
```

Comment:
We decided to always write "db-ally" starting with a lowercase letter but does this also extend to "ragbits"? Looks a little weird to me

Reply:
Changed to capital letter